hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsHttpOperation.java

```diff
@@ -657,4 +653,36 @@ public String getResponseHeader(final String httpHeader) {
       return "";
     }
   }
+
+  public static class AbfsHttpOperationWithFixedResultForGetFileStatus extends AbfsHttpOperation {
```

Comment:
lets add a javadoc about the requriement of new classes.

Reply:
Yes, 
Added